### PR TITLE
Select component in template editor

### PR DESCRIPTION
**Pull request by olegrise**
## Description
Select a component in the template editor.

## Motivation and Context
Implementation of the hackathon project.

## How Has This Been Tested?
Manually and with automated tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/rise-preview.js [hunk 1]
```diff
@@ -19,15 +22,66 @@ RisePlayerConfiguration.Preview = (() => {
...
+      break;
     //defaults to attributData for backwards compatibility
     default:
       RisePlayerConfiguration.AttributeData.update( data );
       break;
     }
   }
 
+  function _postMessageToEditor( type, value ) {
+    var message = {
```
Comment (andrecardoso) [inline src/rise-preview.js]: Can we use `const` here?

File: src/rise-preview.js [hunk 2]
```diff
@@ -19,15 +22,66 @@ RisePlayerConfiguration.Preview = (() => {
...
+      type: type,
+      value: value
+    };
+
+    window.parent.postMessage( JSON.stringify( message ), "*" );
+    console.log( "posted" );
```
Comment (andrecardoso) [inline src/rise-preview.js]: Can we remove this `console.log` call?

File: src/rise-preview.js [hunk 3]
```diff
@@ -19,15 +22,66 @@ RisePlayerConfiguration.Preview = (() => {
...
+      .forEach( element => {
+        element.onclick = ( el ) => {
+          _highlightComponent( el.srcElement.id );
+          _postMessageToEditor( "editComponent", el.srcElement.id );
+        };
+      });
+  }
+
+  function _highlightComponent( id ) {
+    let el = RisePlayerConfiguration.Helpers.getComponent( id );
```
Comment (andrecardoso) [inline src/rise-preview.js]: Can we use `const` here?

File: src/rise-preview.js [hunk 4]
```diff
@@ -19,15 +22,66 @@ RisePlayerConfiguration.Preview = (() => {
...
+
+    if ( el ) {
+      let rect = el.getBoundingClientRect();
```
Comment (andrecardoso) [inline src/rise-preview.js]: Can we use `const` here?

File: src/rise-preview.js (hunk 2 above)
Comment (olegrise) [reply src/rise-preview.js]: Oops ;-). Good catch.

**Pull request merged**